packages/libllvm/build.sh

```diff
@@ -119,16 +117,3 @@ termux_step_post_make_install() {
 		ln -f -s clang-${TERMUX_PKG_VERSION:0:1} $tool
 	done
 }
-
-termux_step_post_massage() {
-	sed $TERMUX_PKG_BUILDER_DIR/llvm-config.in \
-		-e "s|@TERMUX_PKG_VERSION@|$TERMUX_PKG_VERSION|g" \
-		-e "s|@TERMUX_PREFIX@|$TERMUX_PREFIX|g" \
-		-e "s|@TERMUX_PKG_SRCDIR@|$TERMUX_PKG_SRCDIR|g" \
-		-e "s|@LLVM_TARGET_ARCH@|$LLVM_TARGET_ARCH|g" \
-		-e "s|@LLVM_DEFAULT_TARGET_TRIPLE@|$LLVM_DEFAULT_TARGET_TRIPLE|g" \
-		-e "s|@TERMUX_ARCH@|$TERMUX_ARCH|g" > $TERMUX_PREFIX/bin/llvm-config
-	chmod 755 $TERMUX_PREFIX/bin/llvm-config
-	cp $TERMUX_PKG_HOSTBUILD_DIR/bin/llvm-tblgen $TERMUX_PREFIX/bin
-	cp $TERMUX_PKG_HOSTBUILD_DIR/bin/clang-tblgen $TERMUX_PREFIX/bin
```

Comment:
@its-pointless, you added these two, but as above, they're not packaged since they aren't in the massage dir (plus, these are the host binaries built for x86_64, probably a typo). I notice that an AArch64 `llvm-tblgen` is already packaged currently, but `clang-tblgen` isn't installed by the LLVM config by default: would you like me to add it?

Reply:
When we added the fast-build `-i` option we were having problems with packages trying to execute llvm-tblgen and clang-tblgen, the reason for the host binaries being installed here is probably related to that. I think we solved it some other way do so can probably remove it now though

Reply:
Oh, is that why the `llvm-config` script is there too, not for the LLVM packages but for other packages that need to call `llvm-config` on the host linux build server? I just checked and I see that the rust and lldb package build scripts call it.

Reply:
>we need it for dependent packages

No, that's downloaded externally, see the two lines above the one you pasted.

Reply:
Yes, I guess its-pointless put the needed files in a tar and uploaded them to his repo. Maybe updated version is needed for lldb to build with llvm 10

Reply:
this is probably best left until after ndk 21b is released. All the fixes for 21b are completed afaik so its probably in testing. Unless recent events have forced a reschedule at google it should be a week or 2 away.

Reply:
I see no reason to wait, the NDK sysroot and clang are independent. In fact, I was surprised to find yesterday that this repo goes out of its way not to use the NDK sysroot package when cross-compiling Termux packages, unlike for on-device builds.

Reply:
More about bandwidth managemen5

Reply:
More about bandwidth managemen5

Reply:
it becomes a more significant issue since rust will also need to be updated.

Reply:
OK, I don't know how big a deal updating Rust is, I don't use it.

Reply:
Tried updating the LLVM that the Rust package links against, let's see if it works.